diff --git a/node_modules/createsend-node/lib/interface/utils.js b/node_modules/createsend-node/lib/interface/utils.js
index acc6561..b39f481 100644
--- a/node_modules/createsend-node/lib/interface/utils.js
+++ b/node_modules/createsend-node/lib/interface/utils.js
@@ -44,22 +44,20 @@ module.exports = {
                 if (error) { return callback(error); }

                 var code = response.statusCode;
-                if (code == 200 || code == 201 || code == 202) {
-                    if (body) {
-                        try {
-                            callback(null, JSON.parse(body));
-                        } catch (e) {
-                            callback(new Error('Unable to parse JSON.  Error: ' + e + '\nBody:\n' + body));
-                        }
-                    } else {
-                        callback(null);
+                if (body) {
+                    try {
+                        body = JSON.parse(body);
+                        body.headers = response.headers;
+                    } catch (e) {
+                        callback(new Error('Unable to parse JSON.  Error: ' + e + '\nBody:\n' + body));
+                        return;
                     }
+                }
+
+                if (code == 200 || code == 201 || code == 202) {
+                    callback(null, body);
                 } else {
-                    try {
-                            callback(JSON.parse(body));
-                        } catch (e) {
-                            callback(new Error('Unable to parse JSON.  Error: ' + e + '\nBody:\n' + body));
-                        }
+                    callback(body);
                 }
             }
         );
